packages/contracts/docs/osx/01-how-it-works/01-core/03-plugins/index.md

```diff
@@ -29,15 +29,13 @@ Lern more about the different [plugin types](../../../02-how-to-guides/02-plugin
 
 This raises questions on how the DAO manages plugins and who actually owns plugins.
 
-### How does the DAO Manage a Plugin?
+### How Does the DAO Manage a Plugin?
```

Comment:
Where is the issue with "does"?

Reply:
As noted by Michael on [this discussion](https://github.com/aragon/osx/pull/434#discussion_r1266716602), headings should be in title case